akvo/rsr/models/country.py

```diff
@@ -85,7 +85,7 @@ def iati_country(self):
         return codelist_value(codelist_models.Country, self, 'country')
 
     def iati_country_unicode(self):
-        return str(self.iati_country())
+        return unicode(self.iati_country())
```

Comment:
@zzgvh `self.iati_country()` is actually calling `codelist_value`, not `codelist_name`, and therefore returns and instance of `codelists.models.Country`.  The `unicode` call is required.